src/OrbitGl/TimelineUi.cpp

```diff
@@ -35,16 +38,30 @@ void TimelineUi::RenderLabels(Batcher& batcher, TextRenderer& text_renderer,
   const float kLabelMarginBottom = 2;
 
   float previous_label_end_x = std::numeric_limits<float>::lowest();
-  for (uint64_t tick_ns : timeline_ticks_.GetMajorTicks(min_timestamp_ns, max_timestamp_ns)) {
-    // TODO (b/170712621): Test the new format (ISO 8601) to draw timestamps.
-    std::string text = orbit_display_formats::GetDisplayTime(absl::Nanoseconds(tick_ns));
+  std::vector<uint64_t> all_major_ticks =
+      timeline_ticks_.GetMajorTicks(min_timestamp_ns, max_timestamp_ns);
+
+  int max_precision = 0;
```

Comment:
Nit: The name is a bit vague. I assume it's the number of digits after the decimal point that are needed for the timestamps, right?

So what about `number_of_decimal_places_needed`?

(Only a nit, `max_precision` also works!)